From 38d9f2dd08afe94130e56eb2e627a3a6bbc1b5c6 Mon Sep 17 00:00:00 2001
From: Stella Laurenzo <stellaraccident@gmail.com>
Date: Mon, 5 May 2025 18:34:48 -0700
Subject: [PATCH 3/3] Add missing `chrono` include.

Flagged as an error when building with Microsoft's runtime library.
---
 clients/benchmarks/rocsparse_bench_app.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/clients/benchmarks/rocsparse_bench_app.cpp b/clients/benchmarks/rocsparse_bench_app.cpp
index f225a385..862b3043 100644
--- a/clients/benchmarks/rocsparse_bench_app.cpp
+++ b/clients/benchmarks/rocsparse_bench_app.cpp
@@ -25,6 +25,7 @@
 #include "rocsparse_bench_app.hpp"
 #include "rocsparse_bench.hpp"
 #include "rocsparse_random.hpp"
+#include <chrono>
 #include <fstream>
 
 rocsparse_bench_app* rocsparse_bench_app::s_instance = nullptr;
-- 
2.49.0.windows.1

